api/api.proto

```diff
@@ -3454,6 +3531,110 @@ message SdkRoleUpdateResponse {
   SdkRole role = 1;
 }
 
+// SdkFilesystemTrimStartRequest defines a request to start a background filesystem trim operation
+message SdkFilesystemTrimStartRequest {
+  // Id of the volume
+  string volume_id = 1;
+  // Path where the volume is mounted
+  string mount_path = 2;
+}
+
+// SdkFilesystemTrimStartResponse defines the response for a SdkFilesystemTrimStartRequest. If case of error, this response struct is not valid
+message SdkFilesystemTrimStartResponse{
+  // Id of the volume
+  string volume_id = 1;
+  // Status of the background filesystem trim Start request
+  string status = 2;
+}
+
+// SdkFilesystemTrimGetStatusRequest defines a request to get status of a background filesystem trim operation
+message SdkFilesystemTrimGetStatusRequest {
+  // Id of the volume
+  string volume_id = 1;
+  // Path where the volume is mounted
+  string mount_path = 2;
+}
+
+// SdkFilesystemTrimGetStatusResponse defines the response for a SdkFilesystemTrimGetStatusRequest. If case of error, this response struct is not valid
+message SdkFilesystemTrimGetStatusResponse{
+  // Id of the volume
+  string volume_id = 1;
+  // Status of the background filesystem trim GetStatus request
+  string status = 2;
+}
+
+// SdkFilesystemTrimStopRequest defines a request to stop a background filesystem trim operation
+message SdkFilesystemTrimStopRequest {
+  // Id of the volume
+  string volume_id = 1;
+  // Path where the volume is mounted
+  string mount_path = 2;
+}
+
+// SdkFilesystemTrimStopResponse defines the response for a SdkFilesystemTrimStopRequest. If case of error, this response struct is not valid
+message SdkFilesystemTrimStopResponse{
```

Comment:
Why would the stop response return information? Shouldn't it just be error or nil?

Reply:
fixing it to return a status code